### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by pablobm**
As mentioned at https://github.com/code-corps/ember-stripe-elements/pull/50, something has changed that has broken the build. I don't know what, as the latest commit on `develop` passed the build.

In any case, this fixes the build. It involved:

* Fixing some linter warnings.
* Changing the way a mock service is injected into tests. The existing strategy was not supported in Canary.

### CODE REVIEW

File: addon/components/stripe-element.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ export default Component.extend({
   classNames: ['ember-stripe-element'],
 
   autofocus: false,
-  options: {},
+  options: computed(() => ({})),
```
Comment (josemarluedke) [inline addon/components/stripe-element.js]: Why was this change required? Ember has deprecated computed overrides recently and I'm worried that with this change, the addon will now have these deprecations.

Comment (lindyhopchris) [reply addon/components/stripe-element.js]: the change is right for the mo but as you say, isn't very future proof.

Using `this.set('options', {});` within the `init()` method might be better at the mo. Not sure when `init()` will be deprecated though.

Comment (josemarluedke) [reply addon/components/stripe-element.js]: Using `init()` should be safe.

Comment (pablobm) [reply addon/components/stripe-element.js]: Right, I wasn't aware of this. I'm now back on an Ember project after some time away, and it looks like things have changed a bit. I'm happy to do this on `init()` if that's the done thing now.

Comment (lindyhopchris) [reply addon/components/stripe-element.js]: Ah thanks! No problem, there's a number of deprecations in-flight at the mo so understandable if you've been away from Ember for a while!

Comment (pablobm) [reply addon/components/stripe-element.js]: Sorted now! Let me know if this solution is up to snuff :crossed_fingers:

Comment (lindyhopchris) [reply addon/components/stripe-element.js]: Great, thanks!
